platform/macos/detect.py

```diff
@@ -236,10 +237,17 @@ def configure(env: "Environment"):
         ]
     )
     env.Append(LIBS=["pthread", "z"])
+    env.Append(CCFLAGS=["-Wno-deprecated-declarations"])  # Disable deprecation warnings
```

Comment:
Is this still relevant, or was that to compile ANGLE?

I'd prefer to enable it selectively for the `SCsub` or file(s) where it's relevant if we can, so we still get deprecation warnings for things we do need to migrate away from.

Reply:
Actually, it's not necessary. It was added for building ANGLE in place (native GL parts have pragmas in the relevant files). Removed.